src/EnergyPlus/DaylightingManager.cc

```diff
@@ -9465,12 +9360,15 @@ void DayltgInteriorMapIllum(EnergyPlusData &state, int &ZoneNum) // Zone number
     }
 
     if (state.dataGlobal->WarmupFlag) return;
+
     //              Initialize reference point illuminance and window background luminance
 
-    for (ILM = 1; ILM <= state.dataDaylightingData->ZoneDaylight(ZoneNum).MapCount; ++ILM) {
+    for (int mapNum = 1; mapNum <= state.dataDaylightingData->TotIllumMaps; ++mapNum) {
```

Comment:
I like this. SysAvailMangers loop over all equip types regardless whether there is an avail manager. GOOD.